### ISSUE TITLE
Not triggered until user interaction on Ubuntu 15.10

### ISSUE THREAD

**Issue by Koslun** [alirezamirian/angular-chrome-autofill-fix#3]
I might be missing something obvious but still seeing the bug with my setup:

**Chrome** 50.0.2661.102 (64-bit)
**Ubuntu** 15.10
**Angular.js** 1.5.6
**Angular Material** 1.0.9

Installed and added 'chrome-autofill-fix' as a dependency to my app module so to speak.

Tried using both label and placeholder as seen below:

``` HTML
<md-input-container>
     <input type="password" name="password" ng-model="user.password" required 
              placeholder="Password">
</md-input-container>
```

``` HTML
<md-input-container>
    <label>Password</label>
    <input type="password" name="password" ng-model="user.password" required>
</md-input-container>
```

Can also report that the `$log.info` output (`bypassing required validator because of Chrome auto-filling`) isn't triggered until I start interacting with the page.

**Comment by alirezamirian** [comment]
Have you spotted any error in the console?
Can you provide a live example on plunker, codepen, etc?
Isn't there any randomness for working or not working?

**Comment by Koslun** [comment]
No error in the console and no randomness as far as I can tell.

I did however now notice that reloading by pressing "F5" does make the labels float automatically, it does however not trigger the code either so from what I remember reading of the issue thread the bug is not triggered when reloading via "F5".

For the following scenarios waiting five or more seconds the password label does not float up:
- Reloading by entering the address to the site
- Manually pressing the reload button
- Reloading through the audit in chrome developer tools

Don't have a codepen at this point in time. Can see if I can try to modify an Angular Material codepen I suppose.

**Comment by alirezamirian** [comment]
Yes, the problem doesn't show up in all reload scenarios. For me, it always shows up when I hit enter on address bar, but doesn't show up with browser reload button.
Anyway, it would be great if you provide a codepen or something.

**Comment by Koslun** [comment]
Think I have reproduced it here: [codepen editor](http://codepen.io/Koslun/pen/qNOjdQ/?editors=1010), can trigger the autofill if you go into debug mode, like here: [codepen debug](http://s.codepen.io/Koslun/debug/qNOjdQ)

**Comment by Koslun** [comment]
Can add that this version does not appear to work for me on either of the three ways I declared above. Also still using the same rig but codepen uses a bumped version of angular material:

Locally:
**Chrome** 50.0.2661.102 (64-bit)
**Ubuntu** 15.10

The codepen uses:
**Angular.js** 1.5.6
**Angular Material** 1.1.0.rc-5

**Comment by alirezamirian** [comment]
The codepen works fine for me! (I actually added a submit button to it for being able to save the form data first: http://s.codepen.io/alirezamirian/debug/WxQYwo)
my Chrome version is exactly as yours, but I use windows.
Here is how it works for me with and without `chrome-autofill-fix`:

![chrome-autofill-fix2](https://cloud.githubusercontent.com/assets/3150694/15887888/f7e51ed4-2d79-11e6-838a-b420c9254064.gif)

**Comment by Koslun** [comment]
Still not working for me, tried disabling all extensions just in case too. 

Must be something to do with the differences between Windows and Ubuntu versions of Chrome then. Would assume it'd be the same for all Linux users. Would further assume it works on both Windows 7, 8 and 10 but might be best to check that out.

Also curious to know if this is issue persists for Mac OS X users.

**Comment by Koslun** [comment]
EDIT: Can confirm it working on another Windows 10 with Chrome version 50.0 and 

**Mac OSX**: 10.11.5 
**Chrome**: 50.0.2661.102 (64-bit).

**Comment by tilwinjoy** [comment]
It doesn't work the very first time, when we click "Save" on the browser notification and browser fills the data on empty fields, probably because it happens after the timeout period. 

Apart from that I can confirm it works afterwards on Ubuntu 14.04 LTS and Chrome 51.0 ~
